sdks/python/apache_beam/runners/direct/watermark_manager.py

```diff
@@ -143,13 +143,18 @@ def _refresh_watermarks(self, applied_ptransform):
             for consumer in consumers:
               self._refresh_watermarks(consumer)
 
-  def extract_fired_timers(self):
+  def extract_all_timers(self):
```

Comment:
<!--new_thread; commit:2ab61a1a97dda2326dc339f5f9dda40a04a47d9f; resolved:0-->
Can we use the previous method name?  This name is misleading because it isn't we still only extract fired timers.